dapr/proto/runtime/v1/dapr.proto

```diff
@@ -687,6 +708,174 @@ message UnlockResponse {
   Status status = 1;
 }
 
+// SubtleGetKeyAlpha1Request is the request object for SubtleGetKeyAlpha1.
+message SubtleGetKeyAlpha1Request {
+  enum KeyFormat {
+    // PEM (PKIX) (default)
+    PEM = 0;
+    // JSON (JSON Web Key) as string
+    JSON = 1;
+  }
+ 
+  // Name of the component
+  string component_name = 1 [json_name="component"];
+  // Name (or name/version) of the key to use in the key vault
+  string name = 2;
+  // Response format
+  KeyFormat format = 3;
+}
+
+// SubtleGetKeyAlpha1Response is the response for SubtleGetKeyAlpha1.
+message SubtleGetKeyAlpha1Response {
+  // Name (or name/version) of the key.
+  // This is returned as response too in case there is a version.
+  string name = 1;
+  // Public key, encoded in the requested format
+  string public_key = 2 [json_name="publicKey"];
+}
+
+// SubtleEncryptAlpha1Request is the request for SubtleEncryptAlpha1.
+message SubtleEncryptAlpha1Request {
+  // Name of the component
+  string component_name = 1 [json_name="component"];
+  // Message to encrypt.
+  bytes plaintext = 2;
+  // Algorithm to use, as in the JWA standard.
+  string algorithm = 3;
+  // Name (or name/version) of the key.
+  string key = 4;
```

Comment:
```suggestion
  string key_name = 4 [json_name="keyName"];
```
and similar changes in all other places. I believe we are calling it keyName in other places i.e. components-contrib

Reply:
Changed